libs/features/mas/src/mas-element.js

```diff
@@ -148,6 +148,7 @@ export class MasElement {
         if (options) this.options = options;
         this.state = STATE_PENDING;
         this.update();
+        this.log?.debug('Pending:', { element: this.wrapperElement });
```

Comment:
I wonder if this has any performance impact, this it needs to kind of serialize the wrapperElement.